src/aave-v2/MorphoGovernance.sol

```diff
@@ -360,8 +363,10 @@ abstract contract MorphoGovernance is MorphoUtils {
         onlyOwner
         isMarketCreated(_poolToken)
     {
-        if (!marketPauseStatus[_poolToken].isBorrowPaused) revert BorrowNotPaused();
-        marketPauseStatus[_poolToken].isDeprecated = _isDeprecated;
+        Types.MarketPauseStatus storage status = marketPauseStatus[_poolToken];
+        if (!status.isBorrowPaused) revert BorrowNotPaused();
+        if (_isDeprecated && status.isLiquidateBorrowPaused) revert LiquidateBorrowIsPaused();
```

Comment:
I'm not sure if this this check is useful. Setting `_isDeprecated` to true while the liquidation is paused would basically have no effect anyway (liquidations still woudn't be allowed). Plus it could be annoying if we are in a hurry to deprecate the market & unpause liquidations

Reply:
It might just be cleaner but I'm ambivalent tbh. @pakim249CAL @Rubilmax @MathisGD @Tristan22400 thoughts?

Reply:
> Setting _isDeprecated to true while the liquidation is paused would basically have no effect anyway (liquidations still woudn't be allowed).

I don't understand this point. Why would liquidations still not be allowed?

Reply:
If I understand correctly it's because the flag `isLiquidateBorrowPaused` is checked before `isDeprecated`:
https://github.com/morpho-dao/morpho-v1/blob/35246206a8ec8a2691fb5316fc6c0037ea37405b/src/compound/PositionsManager.sol#L494-L508

Reply:
Isn't it why we want to add this check?

I mean if `LiquidateBorrowIsPaused == true` the deprecation is useless. So we want to deprecate only if `LiquidateBorrowIsPaused == false`

Reply:
I agree with Merlin. But I also think that in general these kinds of checks might be overkill considering how often we should need to use these functions.